servers/audio/audio_stream.cpp

```diff
@@ -789,10 +798,13 @@ AudioStreamRandomizer::AudioStreamRandomizer() {
 void AudioStreamPlaybackRandomizer::start(double p_from_pos) {
 	playing = playback;
 	{
-		float range_from = 1.0 / randomizer->random_pitch_scale;
-		float range_to = randomizer->random_pitch_scale;
-
-		pitch_scale = range_from + Math::randf() * (range_to - range_from);
+		// Pitch_scale is multiplicative, so picking a random number for it directly will bias it
+		// towards higher pitches (0.5 is down one octave, 2.0 is up one octave). Therefore,
+		// semitones, which are linear, are used when picking a random pitch.
+		// See: https://github.com/godotengine/godot-proposals/discussions/10238
+		// See: https://pressbooks.pub/sound/chapter/pitch-and-frequency-in-music/
```

Comment:
The comment works for me!
Just a quick note, the way godot usually links issues is like this:

```suggestion
		// GH-10238 : Pitch_scale is multiplicative, so picking a random number for it directly will bias it
		// towards higher pitches (0.5 is down one octave, 2.0 is up one octave). Therefore,
		// semitones, which are linear, are used when picking a random pitch.
		// See: https://pressbooks.pub/sound/chapter/pitch-and-frequency-in-music/
```